src/main/java/com/moodybluez/enterprise/service/EntryServiceStub.java

```diff
@@ -20,6 +20,12 @@ public void saveEntry(Entry entry) {
         moodEntryMap.put(entry.date.getDate(), entry);
     }
 
+    /**
+     *
+     * @param date
+     * @return
+     * The method fetches the entries by date
+     */
```

Comment:
I added Javadoc for classes to let other reviewers know what the method does.

Reply:
Complete the JavaDoc to make it useful.   What are the params?  Returns?